.fixtures.yml

```diff
@@ -0,0 +1,17 @@
+# This file can be used to install module dependencies for unit testing
+# See https://github.com/puppetlabs/puppetlabs_spec_helper#using-fixtures for details
+
+---
+fixtures:
+  repositories:
+    stdlib:
+      repo: git://github.com/puppetlabs/puppetlabs-stdlib.git
+      ref: v6.0.0
+    archive:
+      repo: https://github.com/voxpupuli/puppet-archive.git
+      ref: v1.0.0
+    systemd:
+      repo: https://github.com/camptocamp/puppet-systemd.git
+      ref: 1.0.0
```

Comment:
this is 2.9.0

Reply:
This file is lower bound support, do we only want to support >= 2.9.0 and < 3.0.0?

Reply:
We want to only support the latest major version of a dependency.

Reply:
This is fixed.